script/validators/source_validator.py

```diff
@@ -219,8 +219,35 @@ def check_namespaces(file_path):
     return status
 
 
+def check_includes(file_path):
+    """Checks whether local includes are done via #include<...>"""
+    with open(file_path, "r") as f:
+        path_pattern = re.compile(r'<(.*?)>')
+        includes = os.listdir('src/include')
+        linenum = 0
+        status = True
+        for line in f:
+            linenum += 1
+            if line.startswith('#include <'):
+                path = path_pattern.search(line).groups()[0]
+                path = path.split('/')
+                word = path[0]
+                if word == 'include':
+                    word = path[1]
+                if word in includes:
+                    if status:
+                        LOG.info("Invalid include in %s" % file_path)
+                    status = False
+                    LOG.info("Line %s: %s" % (linenum, line))
+        if status:
+            LOG.info('Use "..." for including local headers.')
+    return status
+
+
 VALIDATORS = [
     check_common_patterns,
+    check_format,
```

Comment:
Don't add this here. `check_format` is already in the list a few lines further down.